pkg/controller/controlplane/valuesprovider.go

```diff
@@ -182,14 +183,6 @@ var (
 		Name: "shoot-system-components",
 		Path: filepath.Join(aws.InternalChartsPath, "shoot-system-components"),
 		SubCharts: []*chart.Chart{
-			{
```

Comment:
Can you elaborate why the rbac is not needed anymore? Even for k8s < 1.25?

Reply:
CCM is being deployed with cluster-admin https://github.com/gardener/gardener-extension-provider-aws/blob/master/charts/internal/shoot-system-components/charts/cloud-controller-manager/templates/rbac-cloud-controller-manager.yaml RBAC so it has all permissions anyway. 

We also don't deploy any separate node-controller so I don't see where these RBACs are used. Are you aware of any specific system usage that I didn't consider ?

Reply:
For a more detailed answer: the RBACs were needed in the past as the controllers use their own service account. Because of https://github.com/kubernetes/cloud-provider/blob/279df9502932e9784fe3a1ac562cbef38e35cbb2/app/controllermanager.go#L404-L406 they now use the node-controller SA which is already deployed as part of the KCM deployment. Only if we find out that the current node-controller RBACs are somehow insufficient we would need to add additional bindings. Even then the target of the binding would be different. So far, I didn't encounter a problem with this change. The same will be for other providers that deploy these RBACs. 

Thanks @rfranzke